samples/subsys/mgmt/updatehub/src/main.c

```diff
@@ -129,6 +129,12 @@ void main(void)
 		LOG_ERR("Error connecting to WiFi");
 		return;
 	}
+#elif defined(CONFIG_MODEM)
```

Comment:
As this is GSM modem specific code, better to have `#elif defined(CONFIG_MODEM_GSM_PPP` here.